apps/settings/src/components/AppList.vue

```diff
@@ -24,6 +24,13 @@
 	<div id="app-content-inner">
 		<div id="apps-list" class="apps-list" :class="{installed: (useBundleView || useListView), store: useAppStoreView}">
 			<template v-if="useListView">
+				<div v-if="showUpdateAll" class="counter">
+					{{ counter }} {{ t('settings', 'apps have an update available') }}
```

Comment:
```suggestion
					{{ t('settings', '{counter} apps have an update available', {counter}) }}
```

Then translators can put the number wherever it makes sense for each language